ChaosMod/Effects/db/Peds/PedsSpawnRoastingLamar.cpp

```diff
@@ -0,0 +1,83 @@
+#include <stdafx.h>
+
+/*
+	Effect by Juhana
+*/
+
+static Ped lamarPed;
+
+static void OnStart()
+{
+	static const Hash model = GET_HASH_KEY("ig_lamardavis");
+
+	Hash relationshipGroup;
+	ADD_RELATIONSHIP_GROUP("_ROASTING_LAMAR", &relationshipGroup);
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, relationshipGroup, GET_HASH_KEY("PLAYER"));
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, GET_HASH_KEY("PLAYER"), relationshipGroup);
+
+	Ped playerPed = PLAYER_PED_ID();
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+
+	lamarPed = CreatePoolPed(4, model, playerPos.x, playerPos.y, playerPos.z, GET_ENTITY_HEADING(playerPed));
```

Comment:
Might make more sense to use the normal `CREATE_PED` instead considering it gets cleaned up by the effect itself afterwards anyways.